biogeophys/FatesHydroWTFMod.F90

```diff
@@ -475,29 +488,32 @@ function psi_from_th_vg(this,th) result(psi)
     ! (note "psd" is the pore-size-distribution parameter, equivalent to "n" from the
     ! manuscript.)
     !
-    ! satfrac = (1._r8 + (alpha*psi)**psd)**(1._r8/psd-1)
+    ! satfrac = (1._r8 + (alpha*psi)**psd)**(-m)
     !
     ! *also modified to accomodate linear pressure regime for super-saturation
     ! -----------------------------------------------------------------------------------
 
+     m = this%m_vg
+     n = this%n_vg
+    satfrac = (th-this%th_res)/(this%th_sat-this%th_res)
     
-    if(th>this%th_max)then
-
-        psi = this%psi_linear_sat(th)
-        
-    elseif(th<this%th_min)then
-        
-        psi = this%psi_linear_res(th)
-
+    ! Junyan disable the linear interperation below
```

Comment:
why disable the linear interoperation?

Reply:
The linear interpretation will result in the > the_sat. I think, instead of disable it, I should leave it as an option.

Reply:
modifications to linear interpolation were omitted, resolving